Backpack/src/main/java/net/skyscanner/backpack/calendar2/data/CalendarCells.kt

```diff
@@ -87,3 +90,25 @@ internal fun CalendarCells(
 
   return CalendarCells(months = months)
 }
+
+private fun daysBeforeRangeStarts(params: CalendarParams): List<LocalDate> {
```

Comment:
this looks like new behaviour compared to the existing calendar - what's the aim for this?

Reply:
The existing calendar fills the gap for the week (e.g. when your range starts with Wednesday) to avoid weird looking beginnings. This, however, is the behaviour used in iOS version of Calendar, so this code unifies it and cutting off this edgecase. It's also easier to implement and test then filling a week gap.